app/views/places/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<main>
+  <div id="place">
+    <picture>
+      <source src="<%= @place.geograph_image_uri %>">
+      <img src="<%= @place.geograph_image_uri %>" alt="<%= @place.title %>" />
+    </picture>
+  </div>
+
+  <div id="license">
```

Comment:
Might be cool if we could markup the license and image with RDFa (e.g. https://opensource.creativecommons.org/ccrel-guide/#Images)

Reply:
Sprint 2 ;) (Do we have that kind of info for the license, or are we operating on the assumption that all of them are the same?)

Reply:
They're all currently CC, but we could probably do something smarter 👍